<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand -->
<html>
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Jason Serviss Projects</title>
    <!-- jquery --><script src="https://code.jquery.com/jquery-3.1.0.min.js" integrity="sha384-nrOSfDHtoPMzJHjVTdCopGqIqeYETSXhZDFyniQ8ZHcVy08QesyHcnOUpMpqnmWq" crossorigin="anonymous"></script>
    <!-- Bootstrap --><link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
    <!-- Font Awesome icons --><link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-T8Gy5hrqNKT+hzMclPo118YTQO6cYprQmhrYwIiQ/3axmI1hQomh7Ud2hPOy8SP1" crossorigin="anonymous">
    <!-- pkgdown --><link href="http://jasonserviss.github.io/css/pkgdown.css" rel="stylesheet">
    <script src="jquery.sticky-kit.min.js"></script>
    <script src="http://jasonserviss.github.io/js/pkgdown.js"></script>
    <!-- mathjax --><script src="https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
     <![endif]-->
  </head>
  <body>
    <div class="container template-home">
<!-- HEADER -->
      <header>
        <div class="navbar navbar-default navbar-fixed-top" role="navigation">
          <div class="container">
            <div class="navbar-header">
              <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
              </button>
              <a class="navbar-brand" href="http://jasonserviss.github.io">Jason Serviss</a>
            </div>
            <div id="navbar" class="navbar-collapse collapse">
              <ul class="nav navbar-nav">
                <li>
                  <a href="http://jasonserviss.github.io/pages/projects.html">Projects</a>
                </li>
                <li>
                  <a href="http://jasonserviss.github.io/pages/contact.html">Contact</a>
                </li>
              </ul>
              <ul class="nav navbar-nav navbar-right"></ul>
              <!--/.navbar-collapse -->
            </div>
            <!--/.container -->
          </div>
          <!--/.navbar -->
        </div>
      </header>
<!-- HEADER FINISHED -->
<!-- MAIN PANEL -->
      <div class="row">
        <div class="contents col-md-9">
          <div id="clustersignificanceextras" class="section level1">
  <!-- ClusterSignificance -->
            <div class="page-header"><h3 class="hasAnchor"><a href="#clustersignificanceextras" class="anchor"></a>ClusterSignificance: a bioconductor package facilitating statistical analysis of class cluster separations in dimensionality reduced data.</h3></div>
            <h4>Description</h4>
            <p>The ClusterSignificance package provides tools to
            assess if class clusters in dimensionality reduced
            data representations have a separation different
            from permuted data. The term class clusters here
            refers to, clusters of points representing known
            classes in the data. This is particularly useful to
            determine if a subset of the variables, e.g. genes
            in a specific pathway, alone can separate samples
            into these established classes. ClusterSignificance
            accomplishes this by, projecting all points onto a
            one dimensional line. Cluster separations are then
            scored and the probability of the seen separation
            being due to chance is evaluated using a permutation
            method.</p>
            <h4>Developers / Authors</h4>
            <p><a href="https://orcid.org/0000-0002-5778-7014">Jason T. Serviss.</a><small class="roles"> Author, maintainer </small></p>
            <p><a href="https://orcid.org/0000-0002-9210-9534">Jesper R. Gådin.</a><small class="roles"> Author </small></p>
            <p><a href="https://orcid.org/0000-0001-7633-403X">Per Eriksson.</a><small class="roles"> Author </small></p>
            <p><a href="https://orcid.org/0000-0003-0708-9530">Lasse Folkersen.</a><small class="roles"> Author </small></p>
            <p><a href="https://ki.se/en/onkpat/granderpokrovskaja-tamm-group">Dan Grandér†.</a><small class="roles"> Author </small></p>
            <h4>Publication</h4>
            <p><a href="https://doi.org/10.1093/bioinformatics/btx393">Oxford Bioinformatics</a></p>
            <h4>Project website</h4>
            <p><a href="https://github.com/GranderLab/ClusterSignificanceExtras">Reproduction of all publication figures.</a></p>
            <h4>Software</h4>
            <p><a href="https://github.com/jasonserviss/ClusterSignificance">ClusterSignificance Website</a></p>
            <br></br>
  <!-- CIM-seq -->
            <div class="page-header"><h3 class="hasAnchor"><a href="#clustersignificanceextras" class="anchor"></a>CIM-seq</h3></div>
            <h4>Description</h4>
            <p>Advances in single-cell biology has enabled us to
            investigate isolated single cells at an
            unprecedented scale and resolution. However, cells
            in multicellular organisms are largely defined by
            their spatial organization within organ structures,
            which means that general methods for studying direct
            cell interaction on a large scale are needed. We
            propose a novel method, Cell Interaction by
            Multiplet Sequencing (CIM-Seq) that uses
            computational deconvolution of RNA-seq data from
            partially dissociated tissue to create cell
            interaction maps. We applied CIM-seq to human fetal
            pancreas, demonstrating that it recapitulates known
            cell interactions such as acinar-ductal cell
            contacts. Furthermore, we discover a strong link
            between a mesenchymal cell subtype and endocrine
            progenitor cells, and identify the set of genes that
            distinguishes this mesenchymal subtype. Thus,
            CIM-Seq is a general method for cell interaction
            studies that can be used on cell types defined to
            an arbitrary resolution allowing identification of
            interacting sub-cell types or cell states.</p>
            <h4>Developers / Authors</h4>
            <p><a href="https://orcid.org/0000-0002-5778-7014">Jason T. Serviss.</a><small class="roles"> Author, maintainer </small></p>
            <p><a href="https://orcid.org/0000-0001-8748-8931">Martin Enge.</a><small class="roles"> Author </small></p>
            <p><a href="https://orcid.org/0000-0003-4139-2904">Nathanael Johansson Andrews.</a><small class="roles"> Author </small></p>
            <h4>Project website</h4>
            <p><a href="https://github.com/jasonserviss/sp.scRNAseq">CIM-seq</a></p>
            <br></br>
<!-- lncTAM34a -->
            <div class="page-header"><h3 class="hasAnchor"><a href="#clustersignificanceextras" class="anchor"></a>An antisense RNA capable of modulating the expression of the tumor suppressor microRNA-34a.</h3></div>
            <h4>Description</h4>
            <p>The microRNA-34a is a well-studied tumor suppressor microRNA
            (miRNA) and a direct downstream target of TP53 with roles in several
            pathways associated with oncogenesis, such as proliferation,
            cellular growth, and differentiation. Due to its broad tumor
            suppressive activity, it is not surprising that miR34a expression
            is altered in a wide variety of solid tumors and hematological
            malignancies. However, the mechanisms by which miR34a is regulated
            in these cancers is largely unknown.  In this study, we find that a
            long non-coding RNA transcribed antisense to the miR34a host gene,
            is critical for miR34a expression and mediation of its cellular
            functions in multiple types of human cancer. We name this long
            non-coding RNA lncTAM34a, and characterize its ability to facilitate
            miR34a expression under different types of cellular stress in both
            TP53 deficient and wild type settings.</p>
            <h4>Developers / Authors</h4>
            <p><a href="https://orcid.org/0000-0002-5778-7014">Jason T. Serviss.</a><small class="roles"> Author, maintainer </small></p>
            <p><a href="https://orcid.org/0000-0003-4139-2904">Nathanael Johansson Andrews.</a><small class="roles"> Author </small></p>
            <p><a href="https://orcid.org/0000-0003-0002-5614">Jimmy Van den Eynden.</a><small class="roles"> Author</small></p>
            <p><a href="https://orcid.org/0000-0002-3415-3449">Felix Clemens Richter.</a><small class="roles"> Author </small></p>
            <p><a href="https://orcid.org/0000-0001-5317-0327">Miranda Houtman.</a><small class="roles"> Author </small></p>
            <p><a href="https://orcid.org/0000-0001-9471-6592">Mattias Vesterlund.</a><small class="roles"> Author </small></p>
            <p><a href="https://orcid.org/0000-0001-9533-692X">Laura Schwarzmueller.</a><small class="roles"> Author </small></p>
            <p><a href="https://orcid.org/0000-0002-5955-0598">Per Johnsson.</a><small class="roles"> Author </small></p>
            <p><a href="https://orcid.org/0000-0003-1400-0119">Erik Larsson.</a><small class="roles"> Author </small></p>
            <p><a href="https://ki.se/en/onkpat/granderpokrovskaja-tamm-group">Dan Grandér†</a><small class="roles"> Author </small></p>
            <p><a href="https://orcid.org/0000-0001-6359-1256">Katja Pokrovskaja Tamm</a><small class="roles"> Author </small></p>
            <h4>Project website</h4>
            <p><a href="https://github.com/GranderLab/miR34a_asRNA_project">lncTAM34a</a></p>
            <h4>Publication</h4>
            <p><a href="https://www.biorxiv.org/content/early/2017/12/15/234310">Biorxiv preprint</a></p>
            <p><a href="https://www.nature.com/articles/s41419-018-0777-5">Cell Death and Disease</a></p>
            <br></br>
  <!-- Acid adapted -->
            <div class="page-header"><h3 class="hasAnchor"><a href="#clustersignificanceextras" class="anchor"></a>A drug screening assay on cancer cells chronically adapted to acidosis.</h3></div>
            <h4>Description</h4>
            <p>Drug screening for the identification of compounds with
            anticancer activity is commonly performed using cell lines cultured
            under normal oxygen pressure and physiological pH. However, solid
            tumors are characterized by a microenvironment with limited access
            to nutrients, reduced oxygen supply and acidosis. Tumor hypoxia and
            acidosis have been identified as important drivers of malignant
            progression and contribute to multicellular resistance to different
            forms of therapy. Tumor acidosis represents an important mechanism
            mediating drug resistance thus the identification of drugs active on
            acid-adapted cells may improve the efficacy of cancer therapy.</p>
            <p>Here, we characterized human colon carcinoma cells (HCT116)
            chronically adapted to grow at pH 6.8 and used them to screen the
            Prestwick drug library for cytotoxic compounds. Analysis of gene
            expression profiles in parental and low pH-adapted cells showed
            several differences relating to cell cycle, metabolism and
            autophagy.
            </p>
            <p>The screen led to the identification of several compounds which
            were further selected for their preferential cytotoxicity towards
            acid-adapted cells. Amongst 11 confirmed hits, we primarily focused
            our investigation on the benzoporphyrin derivative Verteporfin (VP).
            VP significantly reduced viability in low pH-adapted HCT116 cells as
            compared to parental HCT116 cells and normal immortalized epithelial
            cells. The cytotoxic activity of VP was enhanced by light activation
            and acidic pH culture conditions, likely via increased
            acid-dependent drug uptake. VP displayed the unique property to
            cause light-dependent cross-linking of proteins and resulted in
            accumulation of polyubiquitinated proteins without inducing
            inhibition of the proteasome.
            </p>
            <p>Our study provides an example and a tool to identify anticancer
            drugs targeting acid-adapted cancer cells.
            </p>
            <h4>Developers / Authors</h4>
            <p><a href="https://orcid.org/0000-0002-5778-7014">Jason T. Serviss.</a><small class="roles"> Author, maintainer </small></p>
            <p><a href=" https://orcid.org/0000-0003-2591-2914">Angelo DeMilito.</a><small class="roles"> Author </small></p>
            <p><a href="https://liu.se/en/employee/paope93">Paola Pellegrini.</a><small class="roles"> Author </small></p>
            <p><a href="https://orcid.org/0000-0002-8145-7808">Thomas Lundbäck.</a><small class="roles"> Author </small></p>
            <p>Nicolo Bancaro.<small class="roles"> Author </small></p>
            <p><a href="https://www.researchgate.net/profile/Magdalena_Mazurkiewicz3">Magdalena Mazurkiewicz.</a><small class="roles"> Author </small></p>
            <p><a href="https://orcid.org/0000-0003-3988-1784">Iryna Kolosenko.</a><small class="roles"> Author </small></p>
            <p>Di Yu.<small class="roles"> Author </small></p>
            <p><a href="https://ki.se/en/people/mahara">Martin Haraldsson.</a><small class="roles"> Author </small></p>
            <p><a href="https://liu.se/en/employee/padda29">Padraig D’Arcy.</a><small class="roles"> Author </small></p>
            <p><a href="https://orcid.org/0000-0002-4952-445X">Stig Linder.</a><small class="roles"> Author </small></p>
            <h4>Project website</h4>
            <p><a href="https://github.com/GranderLab/acidAdaptedRNAseq">Acid-adapted screening</a></p>
            <h4>Publication</h4>
            <p><a href="https://cancerci.biomedcentral.com/articles/10.1186/s12935-018-0645-5">Cancer Cell International</a></p>
            <br></br>
<!-- STAT3 -->
            <div class="page-header"><h3 class="hasAnchor"><a href="#clustersignificanceextras" class="anchor"></a>STAT3 is activated in multicellular spheroids of colon carcinoma cells and mediates expression of IRF9 and interferon stimulated genes.</h3></div>
            <h4>Description</h4>
            <p>Three-dimensional cell cultures, such as multicellular spheroids
            (MCS), mirror the in vivo architecture of solid tumours and
            multicellular drug resistance. We previously identified interferon
            regulatory factor 9 (IRF9) to be responsible for up-regulation of a
            subset of interferon (IFN)-stimulated genes (ISGs) in MCS of colon
            carcinoma cells. This set of ISGs closely resembled a previously
            identified IFN-related DNA-damage resistance signature (IRDS) that
            was correlated to resistance to chemo- and radiotherapy. In this
            study we found that transcription factor STAT3 is activated upstream
            of IRF9 and binds to the IRF9 promoter in MCS of HCT116 colorectal
            carcinoma cells. Transferring conditioned media (CM) from high cell
            density conditions to non-confluent cells resulted in STAT3
            activation and increased expression of IRF9 and a panel of IRDS
            genes, also observed in MCS, suggesting the involvement of a soluble
            factor. Furthermore, we identified gp130/JAK signalling to be
            responsible for STAT3 activation, IRF9, and IRDS gene expression in
            MCS and by CM. Our data suggests a novel mechanism where STAT3 is
            activated in high cell density conditions resulting in increased
            expression of IRF9 and, in turn, IRDS genes, underlining a mechanism
            by which multicellular drug resistance is regulated..</p>
            <h4>Developers / Authors</h4>
            <p><a href="https://orcid.org/0000-0002-5778-7014">Jason T. Serviss.</a><small class="roles"> Author, maintainer </small></p>
            <p><a href="https://ki.se/en/people/elined">Elin Edsbäcker.</a><small class="roles"> Author </small></p>
            <p><a href="https://orcid.org/0000-0003-3988-1784">Iryna Kolosenko.</a><small class="roles"> Author </small></p>
            <p><a href="https://orcid.org/0000-0003-2236-2099">Caroline Palm-Apergi.</a><small class="roles"> Author </small></p>
            <p><a href=" https://orcid.org/0000-0003-2591-2914">Angelo DeMilito.</a><small class="roles"> Author </small></p>
            <p><a href="https://orcid.org/0000-0001-6359-1256">Katja Pokrovskaja Tamm</a><small class="roles"> Author </small></p>
            <br></br>
  <!-- Pediatric ALL -->
            <div class="page-header"><h3 class="hasAnchor"><a href="#clustersignificanceextras" class="anchor"></a>lncRNA signatures in pediatric ALL.</h3></div>
            <h4>Description</h4>
            <p>Acute lymphoblastic leukemia (ALL) is the most
            common pediatric cancer and remains the leading
            cause of cancer morbidity among pediatric patients.
            ALL can be broadly divided into B- and T-cell
            leukemias and is further sub-classified by
            well-established recurring genetic abnormalities.
            These hallmark features of ALL are utilized
            clinically for treatment stratification but on their
            own cannot account for the observed prognostic
            heterogenecity within ALL or recapitulate the
            disease in in vivo models.</p>
            <p>Comprehensive microarray-based studies have
            previously illustrated that mRNA expression patterns
            correlate well with specific genetic abnormalities
            observed in ALL and were also used to identify new
            subtypes based on gene expression profiling. In
            addition, global transcriptomic and epigenetic
            analysis have uncovered specific signatures related
            to ALL prognosis. Although largely unexplored,
            initial studies have also indicated that lncRNAs may
            be important biomarkers in some ALL subtypes and
            play specific roles in important clinical features,
            such as glucocorticoid resistance.</p>
            <p>By performing strand-specific RNA-seq analysis on
            ALL samples, we will identify functional RNA genes
            that are critical in the oncogenic process and
            further to identify specific targets of these genes.
            Since extensive clinical data is available, this
            will allow us to both analyze how well previously
            defined patient subgroups, such as those with MLL
            and BCR-ABL translocations, are identified when only
            taking into account lncRNAs, and also to define
            whether new prognostic subgroups can be uncovered.
            lncRNA candidates showing promise will subsequently
            be subjected to in-depth bioinformatics-based
            analysis by incorporating additional publically
            available high-throughput datasets. Gene regulation,
            conservation, transcription factor binding, and
            “guilt-by-association” studies will be preformed
            uncovering characteristics and potential functions
            for the lncRNA candidates. In addition, mass
            spectrometry and GRO-seq will be performed on a
            subset of these samples. We will utilize this data
            to correlate lncRNA and protein expression, as well
            as, examining differential expression of actively
            transcribed enhancer RNAs within the ALL subtypes.
            </p>
            <br></br>
          </div>
        </div>
<!-- MAIN PANEL FINISHED -->
<!-- SIDE BAR -->
<!-- I haven't found a great way to align the vertical distance of the sidebar elements -->
<!-- For the moment this is controled via the distance from the top of the document and needs to be manually adjusted -->
<!-- This is done via the div style tag padding-top -->
  <!-- ClusterSignificance -->
        <div class="col-md-3" id="sidebar" style="padding-top: 90px;">
          <h2>License</h2>
          <p><a href="https://www.r-project.org/Licenses/GPL-3">GPL-3</a></p>
          <h2>Status</h2>
          <ul class="list-unstyled">
          <!--Bioconductor General-->
            <li>
              <h4>Bioconductor</h4>
              <a href="http://www.bioconductor.org/packages/devel/bioc/html/ClusterSignificance.html#since"><img border="0" src="http://www.bioconductor.org/shields/years-in-bioc/ClusterSignificance.svg" title="How long since the package was first in a released Bioconductor version (or is it in devel only)."></a>
              <a href="http://bioconductor.org/packages/stats/bioc/ClusterSignificance.html"><img border="0" src="http://www.bioconductor.org/shields/downloads/ClusterSignificance.svg" title="Percentile (top 5/20/50% or 'available') of downloads over last 6 full months. Comparison is done across all package categories (software, annotation, experiment)."></a>
              <a href="https://support.bioconductor.org/t/ClusterSignificance/"><img border="0" src="http://www.bioconductor.org/shields/posts/ClusterSignificance.svg" title="Support site activity, last 6 months: tagged questions/avg. answers per question/avg. comments per question/accepted answers, or 0 if no tagged posts."></a>
              <a href="http://www.bioconductor.org/packages/devel/bioc/html/ClusterSignificance.html#svn_source"><img border="0" src="http://www.bioconductor.org/shields/commits/bioc/ClusterSignificance.svg" title="Average Subversion commits (to the devel branch) per month for the last 6 months"></a>
              <a href="https://codecov.io/gh/jasonserviss/ClusterSignificance"><img src="https://codecov.io/gh/jasonserviss/ClusterSignificance/branch/master/graph/badge.svg" title="Test coverage percentage"></a>
            </li>
            <!--Bioconductor Release-->
            <li>
              <h5>Release</h5>
              <a href="http://www.bioconductor.org/packages/release/bioc/html/ClusterSignificance.html#archives"><img border="0" src="http://www.bioconductor.org/shields/availability/release/ClusterSignificance.svg" title="Whether the package is available on all platforms; click for details."></a>
              <a href="http://bioconductor.org/checkResults/release/bioc-LATEST/ClusterSignificance/"><img border="0" src="http://www.bioconductor.org/shields/build/release/bioc/ClusterSignificance.svg" title="Build results; click for full report"></a>
            </li>
            <!--Bioconductor Devel-->
            <li>
              <h5>Devel</h5>
              <a href="http://www.bioconductor.org/packages/devel/bioc/html/ClusterSignificance.html#archives"><img border="0" src="http://www.bioconductor.org/shields/availability/devel/ClusterSignificance.svg" title="Whether the package is available on all platforms; click for details."></a>
              <a href="http://bioconductor.org/checkResults/devel/bioc-LATEST/ClusterSignificance/"><img border="0" src="http://www.bioconductor.org/shields/build/devel/bioc/ClusterSignificance.svg" title="build results; click for full report"></a>
            </li>
            <!--Travis CI-->
            <li>
              <h4>Travis CI</h4>
              <a href="https://travis-ci.org/jasonserviss/ClusterSignificance"><img src="https://travis-ci.org/jasonserviss/ClusterSignificance.svg?branch=master"></a>
            </li>
          </ul>
        </div>
  <!-- CIM-Seq -->
        <div class="col-md-3" id="sidebar" style="padding-top: 260px;">
          <h2>License</h2>
          <p><a href="https://www.r-project.org/Licenses/GPL-3">GPL-3</a></p>
          <h2>Status</h2>
          <ul class="list-unstyled">
            <!--Travis CI-->
            <li>
              <h4>Travis CI</h4>
              <h5>Release</h5>
              <a href="https://travis-ci.com/jasonserviss/sp.scRNAseq"><img src="https://travis-ci.com/jasonserviss/sp.scRNAseq.svg?token=8VXNvJpmRV9RpFtTcJYW&branch=master"></a>
            </li>
            <li>
              <h5>Devel</h5>
              <a href="https://travis-ci.com/jasonserviss/sp.scRNAseq"><img src="https://travis-ci.com/jasonserviss/sp.scRNAseq.svg?token=8VXNvJpmRV9RpFtTcJYW&branch=devel"></a>
            </li>
            <li>
              <h5>Test coverage</h5>
              <a href="https://codecov.io/gh/jasonserviss/sp.scRNAseq"><img src="https://codecov.io/gh/jasonserviss/sp.scRNAseq/branch/master/graph/badge.svg?token=vVxKKSzhGt" title="Test coverage percentage"></a>
              </ul>
        </div>
<!-- lncTAM34a -->
        <div class="col-md-3" id="sidebar" style="padding-top: 240px;">
          <h2>Containers</h2>
          <ul class="list-unstyled">
          <!--Travis CI-->
            <li>
              <h4>Docker Hub</h4>
              <a href="https://hub.docker.com/r/thegranderlab/mir34a_asrna_project/"><img src="https://img.shields.io/docker/automated/thegranderlab/mir34a_asrna_project.svg?maxAge=2592000?style=plastic"></a>
            </li>
          </ul>
        </div>
  <!-- Acid adapted -->
        <div class="col-md-3" id="sidebar" style="padding-top: 720px;">
          <h2>Status</h2>
          <ul class="list-unstyled">
          <!--Travis CI-->
            <li>
              <h4>Travis CI</h4>
              <a href="https://travis-ci.org/GranderLab/acidAdaptedRNAseq"><img src="https://api.travis-ci.org/GranderLab/acidAdaptedRNAseq.svg"></a>
            </li>
          </ul>
        </div>
      </div>
<!-- SIDE BAR FINISHED -->
<!-- FOOTER -->
      <footer>
        <div class="copyright"><p>Developed by Jason T. Serviss.</p></div>
      </footer>
<!-- FOOTER FINISHED -->
    </div>
  </body>
</html>
